staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/types.go

```diff
@@ -248,3 +340,45 @@ type CustomResourceSubresourceScale struct {
 	// +optional
 	LabelSelectorPath *string
 }
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// ConversionReview describes a conversion request/response.
+type ConversionReview struct {
```

Comment:
We don't use internal ConversionRequests. Should we drop the types?

Reply:
It is the same for `AdmissionReview` as the internal version of that one is also not being used. I will drop it to see if anybody (runtime/compiler/reviewers) complains :)

Reply:
Keep a copy ;-)